plugins/export-workbook/README.md

```diff
@@ -1,9 +1,61 @@
-# @flatfile/plugin-export-workbook
+<!-- START_INFOCARD -->
 
-This package exports data in Flatfile a Workbook. Currently, `.xlsx` is supported.
+The `@flatfile/plugin-export-workbook` plugin exports data in a Flatfile Workbook.
 
-`npm i @flatfile/plugin-export-workbook`
+**Event Type:**
+`listener.on('job:ready')` 
 
-## Get Started
+**Supported file types:**
+`.xlsx`
+
+<!-- END_INFOCARD -->
+
+
+> When embedding Flatfile, this plugin should be deployed in a server-side listener. [Learn more](https://flatfile.com/docs/orchestration/listeners#listener-types)
+
+
+## Parameters
+
+### `debug` *boolean*
```

Comment:
Can we indicated that this is an optional param?

Reply:
fixed